src/main/webapp/donation.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>FeedOn - Donation Form</title>
+
+    <!-- Favicon, CSS & JS -->
+    <link rel="icon" href="assets/logo.png" type="image/png" sizes="16x16">
+    <link rel="stylesheet" href="styles/common.css">
+    <link rel="stylesheet" href="styles/donation-form.css">
+    <script src="js/client.js" defer></script>
+
+    <!-- Font Awesome icons CDN -->
+    <script src="https://use.fontawesome.com/4e51920be5.js"></script>
+
+    <!-- Bootstrap CDN -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+</head>
+
+<body onload="getBlobstoreURL()">
+
+    <form id="donation-form" class="form-container" method="POST" enctype="multipart/form-data">
+
+        <h2 class="center">MAKE A FOOD DONATION</h2>
+
+        <!-- Name -->
+        <label><b>Restaurant Name</b></label>
+        <input placeholder="Enter Restaurant name" name="restaurantName" required>
+
+        <!-- Location -->
+        <label><b>Restaurant Location</b></label>
+        <input placeholder="Enter Restaurant location" name="location" required>
```

Comment:
location is already in the datastore during registration

Reply:
As the user is already logged in, we will know the restaurant's name and location without the user inputting those values again

Reply:
As we had discussed, the autofill of restaurant information in the donation form is a later feature. For MVP we are simply asking the user to enter the details.